src/operator/subgraph/mkldnn/mkldnn_fc.cc

```diff
@@ -298,10 +408,8 @@ void SgMKLDNNFCOp::Forward(const OpContext &ctx,
             full_param_.eltwise_param.scale =
               GetQuantizeScale(output.dtype(), cached_min_output_, cached_max_output_);
           } else {
-            tmp_scale_ =
-              GetQuantizeScale(output.dtype(),
-                               cached_min_output_,
-                               cached_max_output_) / data_scale_;
+          out_scale =  GetQuantizeScale(output.dtype(), cached_min_output_, cached_max_output_);
```

Comment:
It seems tabulation is not correct here somewhere

Reply:
fixed